tests/tests_transactions.py

```diff
@@ -91,6 +91,22 @@ def test_smart_transactions(self):
             with self.assertNumQueries(1):
                 get_category()
 
+    def test_rollback_during_integrity_error(self):
+        # store category in cache
+        get_category()
+
+        # Make current DB be "dirty" by write
+        try:
+            with atomic():
+                Post.objects.create(category_id=-1, title='')
+        except (IntegrityError, Exception):
```

Comment:
`Exception` includes `IntegrityError`. So which on is here?

Reply:
`Exception` was added for debugging test, and left accidentally. Removed it